### PR TITLE
Rebuild /telco

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Rebuild /telco page 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [copy doc](https://docs.google.com/document/d/15FY_IaT39V81FQcoSoEJpmRRjBX80PJ7uxjgzHnDDws/edit)
- Check all the links link to correct pages
- Check design

## Issue / Card

Fixes [#3231](https://github.com/canonical-web-and-design/web-squad/issues/3231)

### CODE REVIEW

File: templates/telco/index.html [hunk 1]
```diff
@@ -278,371 +208,374 @@ <h2 class="p-muted-heading u-no-max-width">OPEN SOURCE COMES THROUGH CONTRIBUTIO
...
+        height="450",
+        width="800",
         hi_def=True,
-        loading="lazy"
+        loading="auto"
         ) | safe
       }}
+      </div>
+      <p>
+        <a href="/openstack/contact-us" class="p-button--positive">Download datasheet</a>
```
Comment (anthonydillon) [inline templates/telco/index.html:330]: This link won't lead to a datasheet

File: templates/telco/index.html [hunk 2]
```diff
@@ -278,371 +208,374 @@ <h2 class="p-muted-heading u-no-max-width">OPEN SOURCE COMES THROUGH CONTRIBUTIO
...
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-9">
+      <h2 class="u-sv3">Network functions management and orchestration with OSM</h2>
+      <p>
+        Supported by 15+ global service providers, the OSM project enables telcos with management and orchestration capabilities for their VNFs and CNFs.
+      </p>
+      <p class="u-sv2">
```
Comment (anthonydillon) [inline templates/telco/index.html:360]: Missing a sentence:
```
Fully operational MANO stack aligned with ETSI NFV. Production-grade, industry-supported, mature.
```

File: templates/telco/index.html [hunk 3]
```diff
@@ -278,371 +208,374 @@ <h2 class="p-muted-heading u-no-max-width">OPEN SOURCE COMES THROUGH CONTRIBUTIO
...
+        Designed for any type of workload: physical, virtual and containerised; open source and proprietary.
       </p>
       <p>
-        <a href="/engage/kata-containers-webinar">
-          Watch the webinar - “Ensuring security and isolation in Kubernetes with Kata Containers”&nbsp;&rsaquo;
-        </a>
+        <a class="p-link--external" href="/engage/charmed-openstack-adoption-whitepaper" class="p-link">Learn more about OSM&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/telco/index.html:364]: Remove external class.

Comment (anthonydillon) [reply templates/telco/index.html:364]: and `p-link` class too

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looks good, few comments inline.

File: templates/telco/index.html (hunk 1 above)
Comment (bethcollins92) [reply templates/telco/index.html]: Wups, I removed the link as in the copy doc it links to another copy doc of a page which hasn't been built yet (correct me if I'm wrong?)

**Review by anthonydillon — APPROVED** [review]
LGTM

**Comment by meralonso** [comment]
Hey @bethcollins92 couple of things 

there's a grey angular shape under the text before the diagram. Not sure if that's for the source file but I can delete that for you
Screenshot 2020-10-14 at 11 37 12

can you left align icons and text in this section, please?
Screenshot 2020-10-14 at 11 37 28

**Comment by meralonso** [comment]
@bethcollins92 can you please replace the NFV stack image for this one? [png](https://assets.ubuntu.com/v1/6921979a-NFVI-vs-stack.png) or [svg](https://assets.ubuntu.com/v1/c1d62d9c-NFVI+vs+stack.svg), whichever you prefer

**Pull request merged**